From 6fa1bdff43d9f20f14cefb03f7ca6d90edc64d1d Mon Sep 17 00:00:00 2001
From: Herrie <Github.com@herrie.org>
Date: Thu, 19 Oct 2023 20:05:49 +0200
Subject: [PATCH] luna-init: Add cust-preferences.txt and
 CustomerCareNumber.txt for luna-syssservice

cust-preferences.txt and CustomerCareNumber.txt are expected by luna-sysservice, add them and install them.

Signed-off-by: Herman van Hazendonk <github.com@herrie.org>
---
Upstream-Status: Inappropriate [LuneOS specific]

 CMakeLists.txt                    | 4 ++++
 files/conf/CustomerCareNumber.txt | 3 +++
 files/conf/cust-preferences.txt   | 4 ++++
 3 files changed, 11 insertions(+)
 create mode 100644 files/conf/CustomerCareNumber.txt
 create mode 100644 files/conf/cust-preferences.txt

diff --git a/CMakeLists.txt b/CMakeLists.txt
index ea8938a..bfd14d7 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -72,3 +72,7 @@ install(DIRECTORY files/conf/fonts DESTINATION ${WEBOS_INSTALL_DATADIR}
 # Can be replaced with a distro-specific version by recipe
 install(FILES files/conf/defaultPreferences.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
     PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
+install(FILES files/conf/cust-preferences.txt DESTINATION ${WEBOS_INSTALL_SYSMGR_DATADIR}/customization
+    PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
+install(FILES files/conf/CustomerCareNumber.txt DESTINATION ${WEBOS_INSTALL_WEBOS_SYSCONFDIR}
+    PERMISSIONS OWNER_READ OWNER_WRITE GROUP_READ WORLD_READ)
\ No newline at end of file
diff --git a/files/conf/CustomerCareNumber.txt b/files/conf/CustomerCareNumber.txt
new file mode 100644
index 0000000..0db3279
--- /dev/null
+++ b/files/conf/CustomerCareNumber.txt
@@ -0,0 +1,3 @@
+{
+
+}
diff --git a/files/conf/cust-preferences.txt b/files/conf/cust-preferences.txt
new file mode 100644
index 0000000..1ee9d5f
--- /dev/null
+++ b/files/conf/cust-preferences.txt
@@ -0,0 +1,4 @@
+{
+	"preferences": {
+	}
+}
